docs/guide/net.rst

```diff
@@ -183,3 +186,19 @@ Finally, if you know the dataset ID of a dataset you wish to download, you can j
 .. code-block:: python
 
     transfer_complete_datasets("AAAAA")
+
+.. _interpolation-keys:
+
+Path interpolation keys
+-----------------------
+
+When downloading DKIST data with ``ds.files.download()``, the ``path=`` keyword argument can be used to specify the target folder for the download.
+This path can include keys corresponding to metadata entries, and those values are then used to complete the download path.
+For example, to download a dataset into its own folder named with the dataset ID, with separate subfolders for each instrument in the dataset, you could set ``path="~/data/dkist/{Dataset ID}/{Instrument}/"``.
+This would take the values for the ``datasetID`` and ``instrumentName`` metadata keywords in the dataset and insert them into the path string.
```

Comment:
```suggestion
For example, to download a dataset into its own folder named with the dataset ID, with separate subfolders for each instrument in the dataset, you could set ``path="~/data/dkist/{dataset_id}/{instrument}/"``.
This would take the values for the dataset ID and instrument name from either the ASDF file or the search results.
```